tests/fuzz/seq_prod_fuzz_example/Makefile

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright (c) Yann Collet, Meta Platforms, Inc.
+ * All rights reserved.
```

Comment:
Typo.
Use # to comment in Makefile instead of /* */

Reply:
Thanks :) copy-pasted without thinking